Core.Auditing/IfxAuditLogger.cs

```diff
@@ -1,101 +1,124 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 
-using Microsoft.Cloud.InstrumentationFramework;
 using Microsoft.CloudMine.Core.Telemetry;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry.Audit.Geneva;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Net.Sockets;
 
 namespace Microsoft.CloudMine.Core.Auditing
 {
+
+    public class TargetResource
+    {
+        public string type;
+        public string name;
+        public string cluster;
+        public string region;
+
+        public TargetResource(string type, string name, string cluster = null, string region = null)
+        {
+            this.type = type;
+            this.name = name;
+            this.cluster = cluster;
+            this.region = region;
+        }
```

Comment:
From what I can see online the standard way to add a request url to an audit-log is to pass it in as its own TargetResource. So this would be captured in the `name` field.